x/lockup/types/msgs.go

```diff
@@ -165,3 +166,33 @@ func (m MsgExtendLockup) GetSigners() []sdk.AccAddress {
 	owner, _ := sdk.AccAddressFromBech32(m.Owner)
 	return []sdk.AccAddress{owner}
 }
+
+var _ sdk.Msg = &MsgForceUnlock{}
+
+// NewMsgBeginUnlockingAll creates a message to begin unlocking tokens.
+func NewMsgForceUnlock(owner sdk.AccAddress, id uint64, coins sdk.Coins) *MsgForceUnlock {
+	return &MsgForceUnlock{
+		Owner: owner.String(),
+		ID:    id,
+		Coins: coins,
+	}
+}
+
+func (m MsgForceUnlock) Route() string { return RouterKey }
+func (m MsgForceUnlock) Type() string  { return TypeMsgBeginUnlockingAll }
+func (m MsgForceUnlock) ValidateBasic() error {
+	_, err := sdk.AccAddressFromBech32(m.Owner)
```

Comment:
Should we also validate id and coins here?

Reply:
good catch! added